Core/Core/Files/FileDetails/FileDetailsViewController.swift

```diff
@@ -366,7 +366,12 @@ extension FileDetailsViewController: PDFViewControllerDelegate {
 
     func saveAnnotations() {
         for child in children {
-            _ = try? (child as? PDFViewController)?.document?.save()
+            if let pdf = child as? PDFViewController {
```

Comment:
Has this not been saving?

Reply:
The delegate method wasn't getting called. I wonder if something changed with the update. I don't know how auto-save is supposed to work but this is the only way I was ever able to trigger the didSave delegate callback. Kinda concerning.